src/styles/components/_grid-example.scss

```diff
@@ -0,0 +1,8 @@
+.grid-example {
```

Comment:
Given that this classname only used to demonstrate the grid system and we don't want to include it in the compiled css, what do you think about moving this to inline styles inside the `GridExample` component?